glue_solar/tests/test_iris_reader.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest.mock as mock
+
+import numpy as np
+import pytest
+from glue_solar.instruments.iris import read_iris_raster
+
+
+DATA = os.path.join(os.path.dirname(__file__), 'data')
+
+
+def test_iris_raster_reader():
+    """
+    To test whether the read_iris_raster is returning a list or a str
+    """
+
+    with mock.patch('glue_solar.instruments.iris', return_value=np.zeros((42, 42, 42))) \
```

Comment:
I can't say I am a fan of having to do `\` due to the line length.

Reply:
Sure, just removed that from the code.